src/lib/web-component/filter-timestamp/filter-timestamp-element.js

```diff
@@ -53,6 +53,30 @@ export class FilterTimestamp extends HTMLElement {
     this.dispatchFilterChange();
   };
 
+  setExistedFilter(timestamp) {
+    if (timestamp == null) {
+      return;
+    }
+
+    this.timeStamp = {};
+
+    if (timestamp?.start != null) {
+      this.timeStamp.start = timestamp.start;
+      this.startTimeLabel.textContent = dateTimeFormat(timestamp.start);
+      this.filterContainer.hidden = false;
+      this.clearStartTimeBtn.hidden = false;
+    }
+
+    if (timestamp?.stop != null) {
+      this.timeStamp.stop = timestamp.stop;
+      this.stopTimeLabel.textContent = dateTimeFormat(timestamp.stop);
+      this.filterContainer.hidden = false;
+      this.clearStopTimeBtn.hidden = false;
+    }
+
+    this.dispatchFilterChange();
```

Comment:
Why is it necessary to call `dispatchFilterChange` here?

Reply:
not needed. Updated :-)